build-system/pr-check.js

```diff
@@ -224,6 +224,7 @@ const command = {
     timedExecOrDie(`${gulp} clean`);
   },
   runLintChecks: function() {
```

Comment:
Maybe rename this to `runLintAndJsonChecks` for implicit documentation in other parts of this file?